app/js/results/rankPlotDirective.html

```diff
@@ -0,0 +1 @@
+<div id="rank-plot"></div>
```

Comment:
dupe id potential

perhaps class is what we want here? Since we're using a restrictive root to select from